editor/store/effects.js

```diff
@@ -452,4 +455,38 @@ export default {
 		const message = spokenMessage || content;
 		speak( message, 'assertive' );
 	},
+	INITIALIZE_META_BOX_STATE( action, store ) {
+		const dataPerLocation = reduce( action.metaBoxes, ( memo, isActive, location ) => {
+			if ( isActive ) {
+				memo[ location ] = jQuery( getMetaBoxContainer( location ) ).serialize();
+			}
+			return memo;
+		}, {} );
+		store.dispatch( metaBoxSetSavedData( dataPerLocation ) );
+	},
+	REQUEST_META_BOX_UPDATES( action, store ) {
+		const dataPerLocation = reduce( getMetaBoxes( store.getState() ), ( memo, metabox, location ) => {
+			if ( metabox.isActive ) {
+				memo[ location ] = jQuery( getMetaBoxContainer( location ) ).serialize();
+			}
+			return memo;
+		}, {} );
+		store.dispatch( metaBoxSetSavedData( dataPerLocation ) );
```

Comment:
Could be [DRY'd](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) up a bit with the `INITIALIZE_META_BOX_STATE` effect logic.

Reply:
Not so simple because the effect is triggering before the action. Didn't you make an update to `refx` to update this?

Reply:
>Not so simple because the effect is triggering before the action. Didn't you make an update to `refx` to update this?

Yes: #3983